backend/tests/Feature/VisitsApiTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Entities\Browser;
+use App\Entities\Demographic;
+use App\Entities\Device;
+use App\Entities\GeoPosition;
+use App\Entities\Os;
+use App\Entities\Page;
+use App\Entities\Session;
+use App\Entities\System;
+use App\Entities\User;
+use App\Entities\Visit;
+use App\Entities\Visitor;
+use App\Entities\Website;
+use DateTime;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class VisitsApiTest extends TestCase
+{
+    use RefreshDatabase;
+
+    private $user;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->user = factory(User::class)->create();
+    }
+
+    public function testPageViewsFilter()
+    {
+        $firstDate = new DateTime('@1565846640');
+
+        $filterData = [
+            'filter' => [
+                'startDate' => $firstDate->getTimestamp(),
+                'endDate' => $firstDate->getTimestamp(),
+                'period' => 60000
+            ]
+        ];
+
+        factory(Website::class)->create();
```

Comment:
Probably better to create entity in factory that depends on another entity. In order to reduce creation of entties to calling just one factory